components/sup/protocols/event.proto

```diff
@@ -98,4 +98,5 @@ message HealthCheckEvent {
   google.protobuf.StringValue stdout = 6;
   // The health check hook stderr output
   google.protobuf.StringValue stderr = 7;
+  google.protobuf.UInt64Value interval = 8;
```

Comment:
Because the healthcheck will always have an interval associated with it, I think it would be better to use the primitive `uint64` type here. This will avoid the need to wrap the value in an `Option` in the rust code. It also might be worth adding a comment stating the units of the interval value.

Reply:
Done!